firestore/integration_test_internal/src/firestore_integration_test.h

```diff
@@ -331,23 +331,24 @@ class FirestoreIntegrationTest : public testing::Test {
   class FirestoreInfo {
    public:
     FirestoreInfo() = default;
-    FirestoreInfo(const std::string& name, UniquePtr<Firestore>&& firestore)
-        : name_(name), firestore_(Move(firestore)) {}
+    FirestoreInfo(const std::string& name,
+                  std::unique_ptr<Firestore>&& firestore)
```

Comment:
Nit: (not added in this PR, but might as well fix here) -- unique pointer is a well-known move-only type, so using an rvalue reference for `firestore` is extraneous (and the style guide kinda frowns upon rvalue references in general). `firestore` can be simply passed by value.